addStudy.py

```diff
@@ -0,0 +1,76 @@
+# !/usr/bin/python3
+
+# addStudy.py
+
+# python addStudy.py -c rwa_configuration.yml -l
+# python addStudy.py -c rwa_configuration.yml -a "studyName"
+
+# Adding new studies to the database
+
+# Import our libraries
+import os
+import sys
+import argparse
+import yaml
+import psycopg2
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "include"))
+from include.calibrationLib import *
+from include.database import *
+
+
+# List all studies
+def get_studies(configuration):
+
+    # return "List to populate"
+    sql = 'SELECT * FROM STUDY'
+    return select(configuration["connection_string"], sql, '')
+
+
+# add studies
+def add_study(configuration, stdName):
+
+    sql = 'INSERT INTO study(Name) VALUES(%s) RETURNING id;'
+
+    return insert_returning(configuration["connection_string"], sql, {'Name': stdName})
+
+
+def main():
+    # Parse the parameters
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-a', action='store', dest='studyname', default=None)
+    # parser.add_argument('-l', action='store', dest='projectname',default=None)
+    parser.add_argument('-l', '--selectq', action='store_true', default=False)
+    parser.add_argument('-c', action='store', dest='conf_file')
+    args = parser.parse_args()
+    insq = args.studyname
+    selq = args.selectq
+
+    try:
+        cfg = load_configuration(args.conf_file)
+
+        if (insq):
+            # insert
+            studies = add_study(cfg, insq)
+            print("Study Id: " + str(studies))
+
+        if (selq):
+            # select
+            rows = get_studies(cfg)
+            print(rows)
```

Comment:
The results can be formatted better than this.

Reply:
The other way of doing it this, but this makes it more complex. 

```
if (insertStudy and not(listStudy)):
            # insert
            studies = add_study(cfg["connection_string"], insertStudy)
            print("Study Id: " + str(studies))
        elif((insertStudy==None or insertStudy=='') and not(listStudy)):
            print("No value passed for insertion, exiting")
            sys.exit(0)

        if (listStudy and insertStudy==None):
            # select
            rows = get_studies(cfg["connection_string"])
            if(len(rows)>=1):
                print(rows)
            else:
                print("Table is empty, exiting")
                sys.exit(0)
```

Reply:
I'm not sure where you are going with that code, nor how it relates to the formatting of the results when you print them to the user.

Reply:
Okay. Got it. I thought you wanted to format the if structure. I'll format the result.